packages/utilities/psammead-test-helpers/index.test.jsx

```diff
@@ -0,0 +1,58 @@
+import * as testHelpers from '.';
+import * as testHelpersFromSrc from './src/index';
+
+const testHelpersExpectedExports = {
+  shouldMatchSnapshot: 'function',
+  shallowRender: 'function',
+  shouldShallowMatchSnapshot: 'function',
+  isNull: 'function',
+  testUtilityPackages: 'function',
+};
+
+const expectedExports = { testHelpers: testHelpersExpectedExports };
+
+const acutalExports = { testHelpers };
+
+const acutalExportsFromSrc = { testHelpers: testHelpersFromSrc };
+
+const testTheTestHelperErrorCases = testHelperMethod => {
+  const consoleLogSpy = jest.spyOn(global.console, 'log');
+
+  const actual = { utility: { foo: {} } };
+  const expected = { utility: { bar: {} } };
+
+  expect(() => {
+    testHelperMethod(actual, expected, 'testing');
+  }).toThrowError();
+  expect(consoleLogSpy).toHaveBeenCalledWith(
+    'No expected export value declared in unit tests for testing/utility.js - foo',
+  );
+
+  consoleLogSpy.mockRestore();
+};
+
+describe('Psammead test helpers', () => {
+  it('should test all the utility exports exist and are the correct type', () => {
+    testHelpers.testUtilityPackages(
```

Comment:
Using one of the test helper functions to test the test helper utility #meta 

![](https://media.giphy.com/media/2fMaOoI1vYA57Ct1cI/source.mp4)